src/import-export/csv-imp/gnc-csv-tokenizer.cpp

```diff
@@ -0,0 +1,77 @@
+#include "gnc-csv-tokenizer.hpp"
+
+#include <iostream>
+#include <fstream>      // fstream
+#include <vector>
+#include <string>
+#include <algorithm>    // copy
+#include <iterator>     // ostream_operator
+
+#include <boost/tokenizer.hpp>
+#include <boost/locale.hpp>
+
+void
+GncCsvTokenizer::set_separators(const std::string& separators)
+{
+    m_sep_str = separators;
+}
+
+
+void GncCsvTokenizer::load_file(const std::string& path)
+{
+    GncTokenizer::load_file_generic(path);
+}
+
+
+int GncCsvTokenizer::tokenize()
+{
+    using Tokenizer = boost::tokenizer< boost::escaped_list_separator<char>>;
+
+    boost::escaped_list_separator<char> sep("\\", m_sep_str, "\"");
+
+    std::vector<std::string> vec;
```

Comment:
If you're going to define an alias (StrVec/str_vec) you should then use it consistently.